keras/saving/save_test.py

```diff
@@ -57,7 +57,7 @@ def assert_h5_format(self, path):
         if h5py is not None:
             self.assertTrue(
                 h5py.is_hdf5(path),
-                "Model saved at path {} is not a valid hdf5 file.".format(path),
+                f"Model saved at path {path} is not a valid hdf5 file.",
```

Comment:
Used f-string format/method in code line 60 for a better and clearer understanding of the code.